TAO/tao/Valuetype/ValueBase.h

```diff
@@ -335,11 +335,11 @@ namespace CORBA
 
   protected:
     DefaultValueRefCountBase ();
-    DefaultValueRefCountBase (const DefaultValueRefCountBase&);
+    DefaultValueRefCountBase (const DefaultValueRefCountBase &) = delete;
     virtual ~DefaultValueRefCountBase ();
 
   private:
-    void operator= (const DefaultValueRefCountBase &);
+    DefaultValueRefCountBase &operator= (const DefaultValueRefCountBase &) = delete;
```

Comment:
I think these also need to be here, but these operators are not explicitly mentioned in the IDL to C++ spec

Reply:
In this case the DefaultValueRefCountBase doesn't have to be derived by the application code, if it doesn't work for users they could use a different class.